enterprise-suite/console_test/setup-minikube.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+minikube start --cpus=3 --memory=6000
+kubectl create serviceaccount --namespace kube-system tiller
+kubectl create clusterrolebinding kube-system:tiller --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+helm init --wait --service-account tiller --tiller-namespace=kube-system
```

Comment:
eol

Reply:
Is this script used anywhere? I think just stick it in the Makefile if you want.

Reply:
It's just something I started using once `install-helm` target disappeared. Yeah, I'll put it in the Makefile.

Reply:
On the other hand... Makefile sets up helm in `lightbend` namespace, which is slightly annoying for testing. Is there a reason why installing Tiller in the good old `kube-system` should be avoided in our test/development clusters?